src/main/java/com/hubspot/jinjava/lib/filter/RegexReplaceFilter.java

```diff
@@ -0,0 +1,50 @@
+package com.hubspot.jinjava.lib.filter;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+
+import com.hubspot.jinjava.doc.annotations.JinjavaDoc;
+import com.hubspot.jinjava.doc.annotations.JinjavaParam;
+import com.hubspot.jinjava.doc.annotations.JinjavaSnippet;
+import com.hubspot.jinjava.interpret.InterpretException;
+import com.hubspot.jinjava.interpret.JinjavaInterpreter;
+
+@JinjavaDoc(
+        value = "Return a copy of the value with all occurrences of a matched regular expression replaced with a new one. " +
+                "The first argument is the regular expression to be matched, the second is the replacement string",
+        params = {
```

Comment:
We should specify which flavor of regex we're supporting here since they differ between python, Java, etc.

Reply:
Add a link to the supported syntax? https://github.com/google/re2/wiki/Syntax